packages/server/database/migrationToCosmos.md

```diff
@@ -0,0 +1,20 @@
+### Possible Solutions
```

Comment:
Thank you for your contribution! Could you add a paragraph at the beginning answering why I would want to migrate my rethink data to mongo?

Reply:
Thanks! paragraph has been added.